backend/src/progress_controller.py

```diff
@@ -2,7 +2,7 @@
 from abc import ABC, abstractmethod
 
 
-class Aborted(Exception):
+class AbortedError(Exception):
```

Comment:
Can we remove whatever rule wants us to add `Error` or `Exception` at the end of all exception classes? I am not a fan of this naming convention because it is useless. Anything that inherits from `Exception` is an error, so why would I need to spell that out again?